src/main/java/core/basesyntax/model/User.java

```diff
@@ -58,4 +58,14 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(login, password, age);
     }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "id=" + id
+                + ", login='" + login + '\''
+                + ", password='" + password + '\''
```

Comment:
Exposing the password through toString is insecure. 
```suggestion
                + ", password='" + password + '\''
```